app/scripts/services/quota.js

```diff
@@ -254,6 +257,84 @@ angular.module("openshiftConsole")
       });
     };
 
+    var OBJECT_COUNT_QUOTAS = [
+      "pods",
+      "replicationcontrollers",
+      "resourcequotas",
+      "services",
+      "secrets",
+      "configmaps",
+      "persistentvolumeclaims",
+      "openshift.io/imagestreams"
+    ];
+
+    var getNotificaitonMessage = function(used, usedValue, hard, hardValue, quotaKey) {
+      var msgPrefix = "Your project is " + (hardValue < usedValue ? 'over' : 'at') + " quota. ";
+      var msg;
+      if (_.includes(OBJECT_COUNT_QUOTAS, quotaKey)) {
+        msg = msgPrefix + "It is using " + usedValue + " of " + hardValue + " " + humanizeQuotaResource(quotaKey, true) + ".";
+      } else {
+        msg = msgPrefix + "It is using " + (usedValue / hardValue * 100) + "% of " + usageWithUnits(hard, quotaKey) + " " + humanizeQuotaResource(quotaKey, true) + ".";
```

Comment:
We should round that percentage to avoid things like repeating decimals. I suggest using the `percent` filter:

https://github.com/openshift/origin-web-console/blob/c6ac85fec6ebff414312f658f467833ee09d3b8b/app/scripts/filters/util.js#L461-L470